float64 time 				
string person_name    	# 交互对象的姓名
string IDtype 
string intention 
float64 p 
string speech 				# 用户的语音内容	
string person_emotion		# 交互对象的情绪
